### PR TITLE
Vocabulary js var

### PR DESCRIPTION
**Pull request by aviv82**
# add a short description of different variable keywords in js 

### General Checks

- [ ] the branch is up to date with `main`/`master`
- [ ] the code works when pulled and run locally
- [ ] All CI checks pass
- [ ] all conflicts are resolved (if any)
- [ ] PR has a descriptive title
- [ ] PR has appropriate labels and milestones for easy identification
- [ ] PR it is assigned to the owner
- [ ] reviewers are assigned
- [ ] the PR contributes only one focused change
- [ ] It is in the appropriate column in the project board (if necessary)
- [ ] has short and clear description
- [ ] is linked to an issue (if it is related)
- [ ] feedback is addressed (if any and if it is appropriate feedback.)

### Markdown

- [ ] the markdown source is formatted
- [ ] spelling and grammar is correct in all text
- [ ] The markdown looks correct when you preview the file
- [ ] all links and images work
- [ ] any embedded code snippets are formatted

### CODE REVIEW

File: vocabulary/javascript/vartypes.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+### var
+
+> var is the og keyword for creating a variable. because of its limitations it
+> is not used as often these days but is still common in 'legacy' (older) code
+
+### let
+
+> let was introduced in 2015 with the 6th edition of ESMAScript (ES6 for short).
+> it is a more versatile keyword than var and is used by developers today
+
```
Comment (arnochauveau) [inline vocabulary/javascript/vartypes.md:31]: Watch out when writing things like this. `var` doesn't have limitations and `let` isn't more versatile. 

In fact, `var` and `let` do exactly the same, but are differently scoped. 
Scoping basically means: "Where are my variables available to me?"
`var` if function scoped and `let` is block scoped.

You can read more about let, const and var here: https://wesbos.com/javascript/01-the-basics/variables-and-statements/#scoping

**Review by arnochauveau — CHANGES_REQUESTED** [review]
See below

**Comment by aviv82** [comment]
> See comment

got it. will edit tonight!

File: vocabulary/javascript/vartypes.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+### var
+
+> var is the og keyword for creating a variable. because it is function scoped
+> it is not used as often these days but still common in 'legacy' (older) code
+
+### let
+
+> let was introduced in 2015 with the 6th edition of ESMAScript (ES6 for short).
+> unlike var it is object scoped and therefore prefered by developers today
```
Comment (arnochauveau) [inline vocabulary/javascript/vartypes.md:30]: It's block scoped, not object scoped. 😃

**Review by arnochauveau — CHANGES_REQUESTED** [review]

File: vocabulary/javascript/vartypes.md (hunk 2 above)
Comment (aviv82) [reply vocabulary/javascript/vartypes.md]: lol woops. 😅 fixed now!

**Review by arnochauveau — APPROVED** [review]

**Pull request merged**